docs/examples/contrib/sqlalchemy/sqlalchemy_declarative_models.py

```diff
@@ -32,15 +33,22 @@ class Book(UUIDAuditBase):
 
 session_config = AsyncSessionConfig(expire_on_commit=False)
 sqlalchemy_config = SQLAlchemyAsyncConfig(
-    connection_string="sqlite+aiosqlite:///test.sqlite", session_config=session_config
+    connection_string="sqlite+aiosqlite:///test.sqlite", session_config=session_config, create_all=True
 )  # Create 'async_session' dependency.
-sqlalchemy_plugin = SQLAlchemyInitPlugin(config=sqlalchemy_config)
 
 
 async def on_startup() -> None:
-    """Initializes the database."""
+    """Initializes the database and adds some dummy data."""
     async with sqlalchemy_config.get_engine().begin() as conn:
         await conn.run_sync(UUIDBase.metadata.create_all)
```

Comment:
```suggestion
```
I think this is what he meant

Reply:
See the commit below, I have indeed solved it by creating the metadata using the parameter, the data via the on_startup.